dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -139,8 +139,7 @@
     private static final Logger logger = Logger.getLogger(DatasetPage.class.getCanonicalName());
 
     public enum EditMode {
-
-        CREATE, INFO, FILE
+        CREATE
```

Comment:
This will be removed in another task?

Reply:
Probably, when we separate creating datasets from main dataset page.